system/keymaps/remote.xml

```diff
@@ -577,16 +577,28 @@
     <remote>
       <left>StepBack</left>
       <right>StepForward</right>
-      <up>ChannelUp</up>
-      <down>ChannelDown</down>
+      <up>Up</up>
+      <down>Down</down>
+      <skipplus>SkipNext</skipplus>
+      <skipminus>SkipPrevious</skipminus>
```

Comment:
no need to remap `skipplus`, `skipminus` because they defaults to `SkipNext` and `SkipPrevious`